src/server/rest/v1/service/RegistrationTokenService.ts

```diff
@@ -145,7 +155,7 @@ export default class RegistrationTokenService {
     UtilsService.assertObjectExists(action, registrationToken, `Registration Token '${tokenID}' does not exist`,
       MODULE_NAME, 'handleDeleteRegistrationToken', req.user);
     // Check auth
-    if (!Authorizations.canDeleteRegistrationToken(req.user, registrationToken.siteArea?.siteID)) {
+    if (!Authorizations.canDeleteRegistrationToken(req.user, registrationToken?.siteArea?.siteID)) {
```

Comment:
No need the first ?, registrationToken is already checked with method assertObjectExists above

Reply:
done I forgot to delete it after re ordering the code